src/components/Scores/InputCounter.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {IconButton, Input} from '@material-ui/core';
+import {AddCircle, RemoveCircle} from '@material-ui/icons';
+
+interface IProps {
+	handleDecrease?: Function;
+	handleChange?: Function;
+	handleIncrease?: Function;
+	value: number;
```

Comment:
1. The same as above - typings
2. Having 'handleIncrease' and 'handleDecrease' alongside 'handleChange' might be too excessive. Both of that methods could be replaced by 'handleChange' with particular values:
```
handleIncrease = handleChange(1);
handleDecrease = handleChange(-1);
```

Reply:
2. These methods have different signature: `handleChange` method uses `React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>` argument to extract numeric value out of it, so this method cannot replace `handleIncrease` and `handleDecrease` (if I understood you correctly)